roles/redpanda_broker/templates/console/defaults.j2

```diff
@@ -0,0 +1,41 @@
+{
+  "kafka": {
+    "brokers": [
+      {% for host in advertised_ips %}
+      "{{ host }}:{{ redpanda_kafka_port }}"{% if not loop.last %},{% endif %}
+      {% endfor %}
+    ]{% if enable_tls | default(false) %},
+  "tls": {
+    "enabled": true,
+    "caFilepath": "{{ redpanda_truststore_file }}",
+    "certFilepath": "{{ redpanda_cert_file }}",
+    "keyFilepath": "{{ redpanda_key_file }}",
+    "insecureSkipTlsVerify": false
+  }
+  {% endif %}
+  },
+  "redpanda": {
+    "adminApi": {
+      "enabled": true,
+      "urls": [
+        {% if enable_tls | default(false) %}
+        {% for host in advertised_ips %}
+        "https://{{ host }}:{{ redpanda_admin_api_port }}"{% if not loop.last %},{% endif %}
+        {% endfor %}
+        {% else  %}
+        {% for host in advertised_ips %}
+        "http://{{ host }}:{{ redpanda_admin_api_port }}"{% if not loop.last %},{% endif %}
+        {% endfor %}
+        {% endif %}
+      ]{% if enable_tls | default(false) %},
+      "tls": {
+        "enabled": true,
+        "caFilepath": "{{ redpanda_truststore_file }}",
+        "certFilepath": "{{ redpanda_cert_file }}",
+        "keyFilepath": "{{ redpanda_key_file }}",
+        "insecureSkipTlsVerify": false
```

Comment:
This is already the default and could probably be ommitted here?

Reply:
I'd prefer to make it explicit since 

* it can be overridden thru the RPCONSOLE variable
* I suspect a non trivial number of users will use self signed certs for testing so I want to remind them by making the setting explicit